neoload/commands/test_results.py

```diff
@@ -213,6 +214,8 @@ def create_json(name, description, quality_status, external_url, external_url_la
         data['externalUrl'] = external_url
     if external_url_label is not None:
         data['externalUrlLabel'] = external_url_label
+    if lock is not None:
+        data['isLocked'] = lock
     if len(data) == 0 and sys.stdin.isatty():
         for field in ['name', 'description', 'qualityStatus', 'externalUrl', 'externalUrlLabel']:
```

Comment:
the "isLocked" option is missing in this list.
This is when the user wants to enter inputs with prompt